python/doc/usecases/use_case_stiffened_panel.rst

```diff
@@ -0,0 +1,207 @@
+.. _use-case-stiffened-panel:
+
+Stiffened panel buckling
+========================
+
+
+Introduction
+------------
+
+The following figure presents a stiffed panel subject to buckling on a military aircraft.
+
+This use-case implements a simplified model of buckling for a stiffened panel, detailed in [ko1994]_.
+
+.. figure:: ../_static/stiffened_panel_illustration.jpg
+    :align: center
+    :alt: buckling illustration
+    :width: 100%
+
+    **Figure 1.** Buckling of a stiffened panel.
+
+
+.. figure:: ../_static/stiffened_panel_simulation.png
+    :align: center
+    :alt: buckling simulation
+    :width: 100%
+
+    **Figure 2.** 3D simulation of buckling.
+
+
+.. figure:: ../_static/stiffened_panel_description.png
+    :align: center
+    :alt: stiffened panel geometry
+    :width: 100%
+
+    **Figure 3.** Parameterization of the stiffened panel.
+
+
+This test case is composed of nine random variables:
+
+- :math:`E\sim\mathcal{TN}(110.0e9, 55.0e9, 99.0e9, 121.0e9)` : Young modulus (Pa)
+
+- :math:`nu\sim\mathcal{U}(0.3675, 0.3825)` : Poisson coefficient (-)
+
+- :math:`h_c\sim\mathcal{U}(0.0285, 0.0315)` : Distance between the mean surface of the hat and the foot of the Stiffener (m)
+
+- :math:`\ell\sim\mathcal{U}(0.04655, 0.05145)` : Length of the stiffener flank (m)
+
+- :math:`f_1\sim\mathcal{U}(0.0266, 0.0294)` : Width of the stiffener foot (m)
+
+- :math:`f_2\sim\mathcal{U}(0.00627, 0.00693)` : Width of the stiffener hat (m)
+
+- :math:`t\sim\mathcal{U}(8.02e-5, 8.181e-5)` : Thickness of the panel and the stiffener (m)
+
+- :math:`a\sim\mathcal{U}(0.6039, 0.6161)` : Width of the panel (m)
+
+- :math:`b_0\sim\mathcal{U}(0.04455, 0.04545)` : Distance between two stiffeners (m)
+
+- :math:`p\sim\mathcal{U}(0.03762, 0.03838)` : Half-width of the stiffener (m)
+
+The output of interest is:
+
+- :math:`(N_{xy})_{cr}`: the critical shear force (N)
+
+We assume that the input variables are independent except the :math:`f_1` and
+:math:`f_2` for which we measure a Spearman correlation of :math:`\rho^S_{12}=-0.8`,
+modelled using a :class:`~openturns.NormalCopula`.
+
+The critical load :math:`(\tau_{xy})_{cr}` of a stiffened panel subject to shear load is given by
+
+.. math::
+  (\tau_{xy})_{cr}=k_{xy}\frac{\pi^2D}{b_0^2t_s}
+
+where
+
+- :math:`a` is the width of the panel;
+
+- :math:`b_0` is the width between too consecutive stiffener feet;
+
+- :math:`t_s` is the thickness of the panel main surface;
+
+- :math:`E_s` is the Young modulus of the panel main surface;
+
+- :math:`\nu_s` is the Poisson coefficient of the panel main surface;
+
+- :math:`D=\frac{E_st_s^3}{12(1-\nu_s^2)}` is the bending coefficient of the
+  panel main surface;
```

Comment:
The equation deserves to be centered, because it is a part of the model and is difficult to read otherwise:
```suggestion
- :math:`D` is the bending coefficient of the
  panel main surface:
.. math:
    D = \frac{E_st_s^3}{12(1-\nu_s^2)};
```